include/inviwo/core/rendering/datavisualizer.h

```diff
@@ -98,11 +98,13 @@ class IVW_CORE_API DataVisualizer {
      * @param filename The file to load in the source processor. The extension of the filename
      *        should be in the list of extensions returned by getSupportedFileExtensions.
      * @param network The network to add the processor to.
+     * @param initialPos the initial position for the processor
```

Comment:
Why "initial"pos? I would "pos" of "offset" not be better?

Reply:
did you consider using a setOrigin method on the visualizer?

Reply:
The processors are positioned at the initialPos and then moved after adding all of them by `util::offsetPosition()`. This results in an animation of the processors moving from the current position to the final one (instead of starting at (0,0)).

Reply:
> did you consider using a setOrigin method on the visualizer?

Yes, I did. Though `addVisualizerNetwork()` and `addSourceProcessor()/addSourceAndVisualizerNetwork()` differentiate in the positioning. `addVisualizerNetwork()` adds stuff relative to the outport, i.e. the initialPos is computed on the fly in each visualizer.

See for example https://github.com/inviwo/inviwo/blob/0148bf08fbbd8d64fe05bb64e089ba84445a53f7/modules/dataframeqt/src/datavisualizer/dataframetablevisualizer.cpp#L87-L91